packages/forms-web-app/src/pages/projects/_middleware/_utils/get-vertical-tabs.js

```diff
@@ -32,6 +32,12 @@ function getVerticalTabs(projectName, caseRef) {
 			name: 'Examination timetable',
 			url: '/projects/' + caseRef + '/examination-timetable'
 		},
+		{
+			hidden: featureFlag.allowHaveYourSay != true && hasOpenTimetables,
```

Comment:
I don't think this is correct. This is saying, if allowHaveYourSay is false and there are open timetables hide the link.

Should it be `! featureFlag.allowHaveYourSay && ! hasOpenTimetables`?